lib/Parser/Parse.cpp

```diff
@@ -11763,6 +11818,10 @@ ParseNodePtr Parser::ConvertArrayToArrayPattern(ParseNodePtr pnode)
         {
             *itemRef = ConvertObjectToObjectPattern(item);
         }
+        else if (item->nop == knopName)
+        {
+            TrackAssignment<true>(item, nullptr, 0, 0);
```

Comment:
They are not used in buildAst code path. So just pass 0 ?

---
In reply to: [67775587](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67775587) [](ancestors = 67775587)